docs/docs/concepts/17.routes.md

```diff
@@ -118,6 +119,17 @@ the `404` action, it would be the equivalent to:
 {path: '404', type: /^.*$/, action: NotFound}
 ```
 
+The Request Matcher is used to define and check that the route path does exist. It avoids potential issues in the logical flow of your bot. To use this new matcher, you can add something like the following:
+
+```javascript
+{
+    path: 'step',
+    request: request =>
+      request.input.data === 'trousers' && request.session.user.id === '1234' && request.lastRoutePath === 'shopping',
+    action: CustomizedSuggestion,
```

Comment:
To fit more in the context `PersonalShopperSuggestion` instead of `CustomizedSuggestion`